### PR TITLE
(#766) Added new walls and tile 

### PR DESCRIPTION
**Pull request by AppleC4T**
- [x] Read and understood [see CONTRIBUTING.md](https://github.com/fossasia/labyrinth/blob/master/CONTRIBUTING.md)
- [x] Included a preview link and screenshot showing after and before the changes.
- [x] Included a description of change below.
- [x] Tried to [squash the commits][squash].

### Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link**
  
  http://rawgit.com/USERNAME/labyrinth/master/index.html

- Fixes https://github.com/fossasia/labyrinth/issues/766

### The problem I want to solve or the facility I want to improve is/are

Add new walls for tiles

### My steps to solve it

Added new wall graphics and replaced a blank tile with a new one containing them in the starter level.

### Screenshots

image

[squash]: https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git

### CODE REVIEW

File: js/tiles.js [hunk 1]
```diff
@@ -317,7 +317,16 @@ const door = {
             alertInfo("Saarthak Chaturvedi Drew this", "Should he become an artist?");
         },
     }),
-    black: Object.assign({}, OpenDoors, {
+	
+	sea: Object.assign({}, OpenDoors, {
```
Comment (niccokunzmann) [inline js/tiles.js]: OpenDoors means, I think, that two doors are open.

File: js/tiles.js [hunk 2]
```diff
@@ -317,7 +317,16 @@ const door = {
...
+		createImages: function(){
+			this.wallTop = this.createImage("tiles/rooms/wall/seasidewall.svg");
+            this.wallRight = this.createImage("tiles/rooms/door/seasidedoor.svg");
```
Comment (niccokunzmann) [inline js/tiles.js]: These are images with a wall and an open door. Thus I think, open doors is not what you would like to use. There should be a closed door.

**Comment by niccokunzmann** [comment]
Hi @AppleC4T, thanks for your contribution.

I added something about the doors. Would you like to change it in this PR?

**Comment by AppleC4T** [comment]
@niccokunzmann what should I change then? Should I change the `canEnterFromTheRight` value?

**Comment by niccokunzmann** [comment]
Can you enter the tile from the right? Can you enter the tile from the top?

**Comment by AppleC4T** [comment]
Yes. You can enter the tile from both right and top.

**Comment by niccokunzmann** [comment]
Why is one wall a wall and one wall a door?

> tiles/rooms/wall/seasidewall.svg
> tiles/rooms/door/seasidedoor.svg

**Comment by niccokunzmann** [comment]
Maybe we should document it a bit better: The tile is rotated by 45° so that the top is actually the topleft.

**Comment by AppleC4T** [comment]
Ok I get, I'll change it so the top has a door as well.

**Comment by AppleC4T** [comment]
Now it should be ok

**Comment by niccokunzmann** [comment]
Cool. You could 

- [ ] rename them for consistency, 
- [ ] create a tile which uses the wall

but that does not keep it from merging.

**Comment by niccokunzmann** [comment]
Thanks, it looks 100% good to me. I will merge now :)

**Pull request merged**